src/env.js

```diff
@@ -26,6 +26,17 @@ export const env = createEnv({
     OPDB_API_TOKEN: z.string(),
     OPDB_API_URL: z.string().url().default("https://opdb.org/api"),
     DEFAULT_ORG_SUBDOMAIN: z.string().default("apc"),
+    // Additional environment variables that were accessed directly via process.env
+    VERCEL_URL: z.string().optional(),
+    PORT: z.string().optional(),
+    // Future API keys mentioned in the issue (optional for now)
+    PINBALL_MAP_API_KEY: z.string().optional(),
+    OPDB_API_KEY: z.string().optional(),
+    // Image storage configuration
+    IMAGE_STORAGE_PROVIDER: z.enum(["local", "vercel-blob"]).default("local"),
+    BLOB_READ_WRITE_TOKEN: z.string().optional(),
+    // Package version for health checks
+    npm_package_version: z.string().optional(),
```

Comment:
So, presumably this was dynamic before, does moving it here cause issues with how the variable was previously used?